src/kata4.humanCatDogYears.js

```diff
@@ -1,5 +1,16 @@
 const humanCatDogYears = (number) => {
+    
+    let firstCatYear = 15;
+    let secondCatYear = 9;
+    let laterCatYears = ((number - 2) * 4);
+    let firstDogYear = 15;
+    let secondDogYear = 9;
+    let laterDogYears = ((number - 2) * 5);
+   
```

Comment:
A couple of things here:
- It's generally better to use `const` over `let`, as it prevents our variables changing into things we don't expect. Especially here as nothing is being re-assigned later.

- If we are using variables as a constant value, we tend to write them in all caps to let other developers know.

- I don't think this code would work if it were passed a 0, 1 or two. You need to add some logic to handle those situations.